docs/cn/compute/Tensorflow.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: global
+title: 在Alluxio-FUSE上运行Tensorflow
+nickname: Tensorflow
+group: Compute Integrations
+priority: 6
+---
+本指南介绍如何在Alluxio POSIX API上运行[Tensorflow](https://www.tensorflow.org/)。
+
+* 目录
+{:toc}
+
+## 概述
+
+Tensorflow使开发人员可以快速轻松地开始使用深度学习。 
+[深度学习]({{ '/en/compute/Deep-Learning.html' | relativize_url}})部分介绍了深度学习的数据挑战 
+以及Alluxio如何帮助解决这些挑战。 
+本教程针对在Alluxio POSIX API之上运行Tensorflow提供一些动手实例和技巧。
+
+## 先决条件
+
+*安装或刷新Java为Java 8 Update 60或更高版本(8u60 +)，64位。
+* Alluxio已设置并好并正常运行。
+*已安装[Tensorflow](https://www.tensorflow.org/install/pip)。本指南使用Tensorflow **v1.15**。
```

Comment:
```suggestion
* 已安装[Tensorflow](https://www.tensorflow.org/install/pip)。本指南使用Tensorflow **v1.15**。
```
Otherwise, markdown will understand the star as italic without the space